docs/js-client-sdk.eppojsclient.getintegerassignment.md

```diff
@@ -16,7 +16,7 @@ getIntegerAssignment(flagKey: string, subjectKey: string, subjectAttributes: Rec
 |  --- | --- | --- |
 |  flagKey | string |  |
 |  subjectKey | string |  |
-|  subjectAttributes | Record&lt;string, any&gt; |  |
+|  subjectAttributes | Record&lt;string, AttributeType&gt; |  |

```

Comment:
I don't get why this is changing - I'm assuming someone else is checking in doc changes without this?